charts/calico/templates/calico-node-rbac.yaml

```diff
@@ -227,6 +270,19 @@ subjects:
 - kind: ServiceAccount
   name: canal
   namespace: kube-system
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: canal-cni-plugin
```

Comment:
Same here - I think if we just keep the name the same (calico-cni-plugin) then we don't need two separate ClusterRoleBindings - one for Canal and one for Calico - we can just use a single one that works for both.